app/services/UserServiceImpl.js

```diff
@@ -29,7 +29,28 @@ module.exports = function(userDao) {
     });
   };
 
+  const findByUsername = function(username, password) {
```

Comment:
Name is changed to `authenticateUser` from suggestion in previous comment. Unsure if the new name is acceptable or not